main.go

```diff
@@ -64,6 +71,29 @@ func initServer() {
 	}
 }
 
+func configJWTMiddleware(router *gin.Engine) {
+	authConfig := config.GetAuth()
+
+	if authConfig.Enabled {
+		ctx := context.Background()
+
+		jwksCache := jwk.NewCache(ctx)
+		err := jwksCache.Register(authConfig.JSONWebKeysEndpoint, jwk.WithMinRefreshInterval(12*time.Hour))
+		if err != nil {
+			log.Fatalf("Failed to register JWKS URL: %v", err)
+		}
+		if _, err := jwksCache.Refresh(ctx, authConfig.JSONWebKeysEndpoint); err != nil {
+			log.Fatalf("URL is not a valid JWKS: %v", err)
+		}
+		fmt.Println("JWKS cache initialized and refreshed")
```

Comment:
Use log.info instead of fmt.Println ?

Reply:
This is outdated. This is no longer in  main.go